tests/core/test_ai_monitoring.py

```
@@ -0,0 +1,265 @@
+"""
+Tests for AI-focused monitoring system.
+"""
+import pytest
+import asyncio
+from datetime import datetime, timedelta
+import random
+from app.core.ai_monitoring import (
+    AIOperationMetrics,
+    RegionalMetrics,
```

Ruff (0.8.2)

`app.core.ai_monitoring.RegionalMetrics` imported but unused

Remove unused import

(F401)